src/remix/templates.ts

```diff
@@ -5,11 +5,7 @@ export const ERROR_BOUNDARY_TEMPLATE_V2 = `const ErrorBoundary = () => {
 };
 `;
 
-export const HANDLE_ERROR_TEMPLATE_V2 = `function handleError(error) {
-  if (error instanceof Error) {
-    Sentry.captureRemixErrorBoundaryError(error);
-  } else {
-    Sentry.captureException(error);
-  }
+export const HANDLE_ERROR_TEMPLATE_V2 = `function handleError(error, { request }) {
+  Sentry.captureRemixServerException(error, 'remix.server', { request });
```

Comment:
@onurtemizkan @AbhiPrasad 

this won't work;

needs to be:
`Sentry.captureRemixServerException(error, 'remix.server', request);`

without `{ }` around `request`

Reply:
Thanks for reporting @wilcoschoneveld! Opened a fix PR. #469